MOBILE_API.xml

```diff
@@ -3160,6 +3165,8 @@
         <param name="heatedMirrorsEnable" type="Boolean" mandatory="false" since="5.0">
             <description>value false means disabled, value true means enabled.</description>
         </param>
+        <param name="climateEnable" type="Boolean" mandatory="false" since="6.0">
```

Comment:
```suggestion
        <param name="climateEnable" type="Boolean" mandatory="false" since="6.0">
            <description>True if the climate module is on, false if the climate module is off</description>
```